src/backend/marsha/core/serializers/video.py

```diff
@@ -449,6 +452,27 @@ def get_lti_url(self, obj):
         )
 
 
+class VideoId3TagsSerializer(VideoBaseSerializer):
```

Comment:
```suggestion
class VideoId3TagsSerializer(serializers.ModelSerializer):
```
I think there is no need to use the base video serializer, if I'm not wrong you are not using it at all.